src/impl/win.cc

```diff
@@ -277,7 +281,23 @@ Serial::SerialImpl::reconfigurePort ()
 void
 Serial::SerialImpl::close ()
 {
-  if (is_open_ == true) {
+  if (is_open_) {
+    // Cancel a blocking operation.
+    DWORD rc;
+    WINBOOL err;
```

Comment:
Not sure where it was defined, but I need to change WINBOOL to BOOL in order to pass the build.

Reply:
Yes, `BOOL` should really be used here instead of `WINBOOL`. Thanks for pointing this out.

```c++
BOOL GetOverlappedResult(
  [in]  HANDLE       hFile,
  [in]  LPOVERLAPPED lpOverlapped,
  [out] LPDWORD      lpNumberOfBytesTransferred,
  [in]  BOOL         bWait
);
```